### ISSUE TITLE
Bower support questionning

### ISSUE THREAD

**Issue by carloshpds** [paradox41/angular-chartist.js#104]
I would like to ask, why drop the bower support if we could transpile the ES2016 code for every release?

**Comment by paradox41** [comment]
Primarily because bower is hardly used anymore. NPM has become the de facto package manager for the web and it is easy enough for bower to install from NPM.

**Comment by carloshpds** [comment]
Thanks for the quick answer @paradox41!

The hardest part to use it, is the ES6 code in projects wich do not use some transpiler. 
I have an entire ecosystem wich do not use ES6.. So, do you recommend to use version `< 4.x`?

**Comment by paradox41** [comment]
So when you install this repo via [these instructions](https://github.com/paradox41/angular-chartist.js#bower), it does not include the compiled assets?

**Comment by carloshpds** [comment]
@paradox41, I've just used:
```
bower install 'https://github.com/paradox41/angular-chartist.js.git#4.2.3' --save
```
..and there's no `dist` folder or any compiled asset. There's a `src` folder wich contains ES2016 code.

![screen shot 2017-01-10 at 12 00 48](https://cloud.githubusercontent.com/assets/2482989/21808910/8284d1ae-d72c-11e6-9642-904767de7e14.png)

**Comment by paradox41** [comment]
Can you try using https://www.npmjs.com/package/bower-npm-resolver and let me know if the `dist` appears there?

**Comment by carloshpds** [comment]
All right, I'll try and let you know it! :)

**Comment by carloshpds** [comment]
@paradox41, 
```
bower install npm:angular-chartist.js chartist angular --save
```
..and there's no `dist` folder :/

**Comment by paradox41** [comment]
Okay. I may have to add the `dist` then. Thanks.

**Comment by paradox41** [comment]
Released as `v4.3.0`

**Comment by carloshpds** [comment]
@paradox41 worked, thanks! 😄
